app/models/patient.rb

```diff
@@ -360,6 +360,13 @@ def latest_transfer
       .distinct
   }
 
+  scope :recently_symptomatic, lambda {
```

Comment:
This seems like all patients who submitted an assessment in the last 60 minutes (not just symptomatic).

Reply:
Yeah, that sounds right. How do we determine if a patient is specifically symptomatic? Looking at the schema, I see patient.status, patient.latest_fever_or_fever_reducer_at, and patient.latest_positive_lab_at as possible options and I'm not sure which is the right one.